docs/development-process/standard-release-process.md

```diff
@@ -0,0 +1,53 @@
+# Standard Release Process
+
+The following document is a description of the standard release process of
+Zarhus-related images. Precise steps and differences from the standard
+process will be described in layer-specific documentation.
+
+## Process steps
+
+1. Make sure that everything that should go into the given release is merged to
+   into main.
+2. Bump `DISTRO_VERSION` in the `conf/distro/<distro-name>.conf` file.
+3. Fill up the `CHANGELOG.md` file with latest changes.
+    - Run `generate-changelog.sh` to generate the change notes for your release.
+    - Use the `generate-changed-recipes` script to print the list of changes in
+      recipe versions. The script uses build manifests - lists of recipes
+      contained in the image along with their versions. You must provide your
+      manifest and the current release's manifest as arguments to the script.
+      The latter can be found in the latest release's section in the GitHub
+      release pages.
+    - Paste the script output into your release's section in `CHANGELOG.md`.
+      Additionally, you must explain why those changes were made and what they
+      introduce (unless the commit messages listed by `generate-changelog.sh`
+      already explain that).
+    - Describe any additional changes.
+4. Commit and push the changelog.
+5. Create and push tag that matches the newly bumped version.
+6. Publish the release on GitHub. Make sure to upload the manifest.
+
+## Versioning scheme
+
+`Zarhus`-related layers are versioned using the
+[semantic versioning](https://semver.org) scheme with the following
+clarifications and additions:
+
+- `PATCH` version is incremented with each released `HOTFIX`
+- `PATCH` version is zeroed with the increment of `MINOR` or `MAJOR` versions
+- `MINOR` version is incremented with every `FEATURE` release (may include more
+  than one feature) unless `MAJOR` version is incremented
+- `MINOR` version is zeroed with the increment of `MAJOR` version
+- `MAJOR` version is incremented according to the project road map. Project road
+  map should define at which point in time or with which feature set next
+  `MAJOR` release is ready.
+
+Depending on the need of a particular software project, a `pre-release`
```

Comment:
@PLangowski, maybe a third-level header? Like `### Release candidate`. Right now, we have configured mkdocs to support 3-level ToC for documents, visible on the right.